src/main/java/wtf/meier/tariff/interpreter/model/Interval.kt

```diff
@@ -1,8 +1,15 @@
 package wtf.meier.tariff.interpreter.model
 
+import kotlinx.serialization.Serializable
+import wtf.meier.tariff.interpreter.model.tariff.TimeBasedTariff
 import java.util.concurrent.TimeUnit
 
+@Serializable
 data class Interval(
     val timeAmount: Int,
     val timeUnit: TimeUnit
-)
\ No newline at end of file
+) {
+    operator fun minus(start: Interval):Interval {
```

Comment:
I am unsure if we should have the operator in the class directly or if we should add it as an extension.

The idea for adding it as an extension is that we separate API representation and the actual logic. What is your opinion here?

Reply:
That's right, I hadn't thought about it. I'll put that in the extensions.